### PR TITLE
Internet Explorer Fixes

### PR DESCRIPTION
**Pull request by dahlbyk**
Some fixes toward https://github.com/huboard/huboard/issues/20 https://github.com/huboard/huboard/issues/640

Hoping @JBertling can give me a hand here; my IE-fu is limited.

### CODE REVIEW

File: ember-app/app/initializers/sockets.js [hunk 1]
```diff
@@ -72,9 +72,7 @@ export function initialize(container, application){
         return channel.replace(/\./g, '!').toLowerCase();
       },
       subscribeTo: function(channel) {
-        const channel = this._sanitizeChannel(channel);
-
-        self = this;
+        const channel = this._sanitizeChannel(channel), self = this;
```
Comment (discorick) [inline ember-app/app/initializers/sockets.js]: Whoa interesting, was self undefined in IE11 ?

Comment (dahlbyk) [inline ember-app/app/initializers/sockets.js]: > Variable undefined in strict mode

**Comment by dahlbyk** [comment]
> Allow Ctrl-Enter (standard on Windows, vs Meta-Enter)

This is confirmed to work for commenting. I thought I remembered someone saying Meta-Enter should also open the Detail screen when creating issues, too, but that's not working for me.

**Comment by dahlbyk** [comment]
I'm kind of tempted to :ship: this as-is to at least make IE not completely broken sooner rather than later. Can revisit with more polish at a later date. Thoughts?

**Comment by discorick** [comment]
Looks good to me!

**Pull request merged**